openlcsd/flow/tasks.py

```diff
@@ -1070,7 +1070,8 @@ def fork_specified_type_imports(
             cli.headers['Authorization'].split()[-1],
             context['config']['TOKEN_SECRET_KEY']
         ),
-        'token_sk': context['config']['TOKEN_SECRET_KEY']
+        'token_sk': context['config']['TOKEN_SECRET_KEY'],
+        'priority': context['priority']
```

Comment:
`priority` can use the default `low`, so does that mean we cannot give this priority setting?  if yes, context['priority'] will raise error

Reply:
The /rest/v1/sources/import/ API support priority parameter, u can see it in https://github.com/RedHatProductSecurity/openlcs/pull/134/files#diff-f51f75ba757bc28435c39e7fd952bb7091214f95ebc69ff662682e64e87c376eR220

priority parameter has a default value `low` if we don't pass the parameter explicitly. Otherwise it will use the value we pass to API. Sub task will inherit parent task's priority as we discussed last week.

Reply:
What will happen if we don't provide this `priority` when importing? because it is  **OPTIONAL**

Reply:
If don't provide `priority` parameter, it's value will set to 'low', cause i set a default value in API(https://github.com/RedHatProductSecurity/openlcs/pull/134/files#diff-619a42fa76805687d3ba147eb55d5fe502fac9cd2980878f218218dc1987fdb6R188). The task and it's sub task will be low priority.

Reply:
It is better to verify it, your setting is different with `license_scan` and `copyright_scan`